views/settings.html

```diff
@@ -218,8 +218,6 @@ <h2 class="nav-tab-wrapper" id="mailpoet_settings_tabs">
       });
     });
     var mailpoet_in_app_announcements = <%= json_encode(settings.in_app_announcements) %>;
```

Comment:
I think it should be safe to remove also this variable and also default value in settings controller https://github.com/mailpoet/mailpoet/blob/51a1187439eb9906c820843900a8107bb484eec9/lib/Settings/SettingsController.php#L66